src/util.cpp

```diff
@@ -568,7 +568,6 @@ const boost::filesystem::path &GetDataDir(bool fNetSpecific)
     }
     if (fNetSpecific)
         path /= BaseParams().DataDir();
-
```

Comment:
Add the newline back in (the change is unnecessary in the diff)

Reply:
added